### PR TITLE
Merge unstable into broken-stdlib

### PR DESCRIPTION
**Pull request by janondrusek**

### CODE REVIEW

File: include/unifex/let_error.hpp [hunk 1]
```diff
@@ -33,89 +34,96 @@
...
+  // time `set_value()` returns `op_` might as well be already destroyed,
+  // without proper deactivation of `op_->sourceOp_`.
   template(typename... Values)
     (requires receiver_of<Receiver, Values...>)
-  void set_value(Values&&... values) noexcept(
+  void set_value(Values... values) noexcept(
       is_nothrow_receiver_of_v<Receiver, Values...>) {
     UNIFEX_ASSERT(op_ != nullptr);
+    unifex::deactivate_union_member(op_->sourceOp_);
     unifex::set_value(std::move(op_->receiver_), (Values&&)values...);
```
Comment (ispeters) [inline include/unifex/let_error.hpp:90]: I wish this said `std::move(values)...`; this is a `move` case, not a forward case.

**Review by ispeters — APPROVED** [review]
LGTM

File: include/unifex/let_error.hpp (hunk 1 above)
Comment (janondrusek) [reply include/unifex/let_error.hpp:90]: Addressed in #450

File: doc/api_reference.md [hunk 2]
```diff
@@ -1267,3 +1298,21 @@ namespace unifex
   };
 }
 ```
+
+### `variant_sender`
```
[the remark this answers is not in the document]
Comment (janondrusek) [reply doc/api_reference.md:1302]: #451

**Pull request merged**